action.yml

```diff
@@ -261,7 +278,7 @@ runs:
                   \"runId\": \"$CLOUDBEES_RUN_ID\",
                   \"runAttempt\": \"$CLOUDBEES_RUN_ATTEMPT\",
                   \"name\": \"$image_name\",
-                  \"url\": \"$url\",
+                  \"url\": \"$info\",
```

Comment:
:white_check_mark: container image artifact info deployment record looked good. There was only one from the local chart example-deployment which is only deployed once.
I am assuming type = `DEPLOYED` is a temporary code till we have the streaming API support for deployed artifacts.